client-side/src/stories/verticalTabs/verticalTabs.scss

```diff
@@ -0,0 +1,48 @@
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
+@import './style.scss';
+.genericButton{
+    font-family: "Nanum Gothic", sans-serif;
```

Comment:
the following comments are for all this file

1. font-family and colors should come from a variable
2. don't use px's
3. use scss structure

Reply:
the code was not changed currently